airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGeneratorIntegrationTest.java

```diff
@@ -212,26 +205,24 @@ protected void insertFinalTableRecords(final boolean includeCdcDeletedAt, final
             """));
   }
 
+  private String dollarQuoteWrap(JsonNode node) {
+    if (node == null) {
+      return "NULL";
+    }
+    final String stringContents = node.isTextual() ? node.asText() : node.toString();
+    // Use dollar quotes to avoid needing to escape anything
+    return StringUtils.wrap(stringContents, "$$");
```

Comment:
meanwhile, in another PR https://github.com/airbytehq/airbyte/pull/29685/files#diff-f668de0cb12e6db0bb67714b015831f7df91559242f22a5d637b864c566dbc12R233

<.<

![image](https://github.com/airbytehq/airbyte/assets/5741425/ea8fa82f-9301-4435-94af-a5776c4b9f89)

Reply:
Should we change this to
```wrap(stringContents, "\\$\\$")```
?

Reply:
works for me!